compiler/luci/pass/src/QuantizationUtils.cpp

```diff
@@ -31,6 +31,36 @@ uint8_t fp32_to_uint8_cast(float f)
   return static_cast<uint8_t>(f);
 }
 
+void asymmetric_wquant_with_minmax_per_layer(CircleConst *node, float min, float max,
```

Comment:
Could you please write a brief comment what this function is supposed to do? I coulnd't grasp it from the name.

Reply:
This function was used to quantize weights in `QuantizeDequantizeWeightsPass.cpp`. It can be used to quantize a constant tensor, so I moved the function to `QuantizationUtils.h` and `QuantizationUtils.cpp` to use it here. 

I've added a brief comment on the function.